### ISSUE TITLE
Add descriptive event title to Dynatrace Integration

### ISSUE THREAD

**Issue by kyle-ssg** [SolidStateGroup/bullet-train-api#2414]
It was noted by Dynatrace that all of our events were just titled Deployment, it would be much better to have the feature name in the title at least.

<img width="1070" alt="image" src="https://github.com/Flagsmith/flagsmith/assets/8608314/237cf39c-65c9-44a6-a4b7-028b9d10cc3e">

https://www.dynatrace.com/support/help/platform-modules/cloud-automation/release-monitoring/version-detection-strategies#events-ingestion

**Comment by dabeeeenster** [comment]
@matthewelwell is this possible currently ?

**Comment by matthewelwell** [comment]
No, but I started on this last night: https://github.com/Flagsmith/flagsmith/pull/2424

**Comment by matthewelwell** [comment]
Just needs wrapping up / testing

**Comment by matthewelwell** [comment]
Resolved in #2424
